test/test_listen_handler.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
```

Comment:
@JenySadadia Please, add another author header with Alex's name and e-mail Alexandra Pereira alexandra.pereira@collabora.com

Reply:
Under a new section `Copyright (C) 2022 Collabora`